aten/src/ATen/gen.py

```diff
@@ -358,38 +360,43 @@ def generate_storage_type_and_tensor(backend, density, scalar_type, declarations
         top_env['cuda_type_headers'].append(
             '#include "ATen/{}.h"'.format(env['Type']))
 
-    return env
```

Comment:
Oh? What happened here?

Reply:
seems like these returns were used for something else in the past but aren't really needed anymore